libs/providers/launchdarkly-client/README.md

```diff
@@ -59,6 +59,16 @@ await OpenFeature.setContext({ key: 'my-key' });
 
 Read more about LD contexts [here](https://launchdarkly.github.io/js-client-sdk/interfaces/LDContextCommon.html)
 
+## Tracking
+
+You can send custom events to LaunchDarkly metrics for use in
+experiments and guarded rollouts. To learn more, read [Metrics](https://launchdarkly.com/docs/home/observability/metrics).
```

Comment:
Shouldn't this link to tracking documentation?

Reply:
I feel like we could link to either https://launchdarkly.com/docs/sdk/features/events or https://launchdarkly.com/docs/home/observability/metrics, but you're right, the events page probably makes more sense.